service/gcs/bridge/connection.go

```diff
@@ -100,28 +70,30 @@ func sendResponseBytes(conn transport.Connection, messageType prot.MessageIdenti
 // stdio pipe to be used. If CreateStd*Pipe for a given pipe is false, the
 // given Connection is set to nil. After creating each transport.Connection, it
 // calls Connect on it.
-func createAndConnectStdio(tport transport.Transport, params prot.ProcessParameters, settings prot.ExecuteProcessVsockStdioRelaySettings) (*StdioConnSet, error) {
-	var err error
-	var stdin transport.Connection
+func createAndConnectStdio(tport transport.Transport, params prot.ProcessParameters, settings prot.ExecuteProcessVsockStdioRelaySettings) (s *stdio.ConnectionSet, err error) {
```

Comment:
What do you think of just "connectStdio"? "create" doesn't seem like the right verb to me.